### ISSUE TITLE
Solid Color Hold Notes 1.0

### ISSUE THREAD

**Issue by HiCube64** [ShadowMario/FNF-PsychEngine#15874]
### Describe your problem here.

I'm trying to change the transparency of the hold notes (or sustain notes) but I can't find a way

### Are you modding a build from source or with Lua?

Source

### What is your build target?

Windows x64

### Did you edit anything in this build? If so, mention or summarize your changes.

No

**Comment by HiCube64** [comment]
I want to do this in psych 1.0

**Comment by Shadow06735** [comment]
you can just edit the sustain line in the note sprite sheet to be more transparent

**Comment by Shadow06735** [comment]
> In the spritesheet it is not transparent, I think it is related to the sourcecode

u can use a photo editing software or website to make the sustain lines transparent

**Comment by HiCube64** [comment]
I just want to make the hold notes opaque like the V-Slice Ones

**Comment by Shadow06735** [comment]
> I just want to make the hold notes opaque like the V-Slice Ones

yeah but i said that doing so with a photo editing software would make things easier

also idk if there is a way to change the sustain line opacity since the notes use a single spritesheet
